.eslintrc.js

```diff
@@ -116,7 +116,10 @@ module.exports = {
             }
           }
         ],
-        'vue/multiline-html-element-content-newline': 'error'
+        'vue/multiline-html-element-content-newline': 'error',
+        'vue/multi-word-component-names': 'off',
+        // This rule gives false positives with asyncData
+        'vue/no-dupe-keys': 'off'
```

Comment:
Just tried building and testing with these removed (ie the rules being applied and not offed) and it worked. Is there a reason for removing them that I missed?

Reply:
Build works, it's linter that complains :)